src/components/Filter/Filter.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export default function FilterRace({ setRace }) {
+  return (
+    <div>
+      <select onChange = {(e)=> setRace(e.target.value)}>
+        <option value="All">All</option>
+        <option value="Dwarf">Dwarf</option>
+        <option value="Elf">Elf</option>
+        <option value="Hobbit">Hobbit</option>
+        <option value="Human">Human</option>
+        <option value="Maiar">Maiar</option>
+        <option value="Orc">Orc</option>
+      </select>
+    </div>
```

Comment:
Nice job passing props and creating the onChange handler. For consistency, remember to keep the name of the function the same as the file name for easier accessibility.